build.sbt

```diff
@@ -0,0 +1,29 @@
+import Dependencies._
+
+ThisBuild / scalaVersion     := "2.13.4"
```

Comment:
Hmm pls stick to Scala 2.12 ;)

Reply:
I mean, we can ofc cross-compile to 2.13 as well... that shouldn't difficult, the most significant difference is about lazy collections (`.view` & co.). But still, given the circumstances, 2.12 support is the priority ;)